htdocs/product/card.php

```diff
@@ -1083,10 +1089,47 @@
 
 		// Batch number management
 		if (!empty($conf->productbatch->enabled)) {
-			print '<tr><td>'.$langs->trans("ManageLotSerial").'</td><td colspan="3">';
+			print '<tr><td>'.$langs->trans("ManageLotSerial").'</td><td>';
```

Comment:
All the lines before are using 4 columns.
You can remove the colspan to allow space for the td of batch number selection. But you must also print the <td></td> if we are no in the case the selection must be shown (so add a ampty td into the else for example)